locales/en.json

```diff
@@ -65,26 +35,23 @@
   "landing-page.steps-recruit": "how you're planning to recruit participants",
   "landing-page.steps-tools": "which tools you'll be using (e.g. a voice recorder, notepad or camera)",
   "landing-page.steps-activities": "the activities you'll be asking the participant to do",
-  "landing-page.leadin": "Choose from a:",
   "landing-page.option1": "Make a consent form for intercept recruitment and testing",
   "landing-page.option2": "Make a form for recruiting through a third party",
-  "template-select.title": "",
-  "template-select.intro": "",
-  "template-select.intercept": "",
-  "template-select.intercept.desc": "",
-  "template-select.thirdparty": "",
-  "template-select.thirdparty.desc": "",
-  "template-select.onlinesurvey": "",
-  "template-select.onlinesurvey.desc": "",
+  "template-select.title": "template-select.title",
+  "template-select.intro": "template-select.intro",
+  "template-select.intercept": "template-select.intercept",
+  "template-select.intercept.desc": "template-select.intercept.desc",
+  "template-select.thirdparty": "template-select.thirdparty",
+  "template-select.thirdparty.desc": "template-select.thirdparty.desc",
+  "template-select.onlinesurvey": "template-select.onlinesurvey",
+  "template-select.onlinesurvey.desc": "template-select.onlinesurvey.desc",
```

Comment:
We can probably delete these template-select strings? Maybe we used to use them and now don't? (and in the French file as well)

Reply:
@sastels It is the beginning of a new page to select templates, a long way off though... I just sneaked it in with the hope that I'd have more time to do more work on it,